lms/djangoapps/mobile_api/users/views.py

```diff
@@ -209,6 +211,10 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
         Get information about the courses that the currently signed in user is
         enrolled in.
 
+        Courses that have expired (because of audit enrollment limitations)
```

Comment:
I'd probably say something like content gating course duration limitations since content gating is the feature and course duration is the app name. or at least I would include content gating

Reply:
ok